_includes/version.html

```diff
@@ -1,6 +1,8 @@
 <div class="version-marker">
   <p>
     Latest versions:
+      <a href="{{ site.baseurl }}manuals/2.5/index.html#Releasenotesfor2.4">2.5.0</a>
+      /
       <a href="{{ site.baseurl }}manuals/2.4/index.html#Releasenotesfor2.4">2.4.0</a>
       /
       <a href="{{ site.baseurl }}manuals/2.3/index.html#Releasenotesfor2.3">2.3.4</a>
```

Comment:
Not sure if I should remove 2.3.4 as we decided to do 2.3.5

Reply:
Since we're going to do a 2.3.5, I'd keep it in for now.